std_msgs/Header header
extended_object_detection/ComplexObject[] complex_objects
